src/index.js

```diff
@@ -40,12 +40,24 @@ const normalizeId = (id) => {
   return id;
 };
 
+const getEntries = ({ entries }) => {
+  if (!entries) {
```

Comment:
I'd negate that to always return an empty array if no condition hits ... for example:

> const getEntries = (options) => {
>   if(options.entries) {
>     return options.entries;
>   } else {
>     //fallback to v1 
>     return Object.keys(entries).map(key => ({ find: key, replacement: entries[key] }));
>   }
>   return [];
> };